### DISCUSSION TITLE
Pi Zero 2W, and 17XX REU compatibility?

### DISCUSSION THREAD

**Discussion by Ratteler** [frntc/Sidekick64#65] [General]
Has any one tried a Pi Zero 2W with the Sidekick? It is essentially a Pi3 in a smaller form factor.
Also, is there any chance of C= 17xx REU compatibility like the RAD Expansion board.

It feels like "Sofie's Choice" choosing between REU compatibility on the RAD, and essentially everything else on the Sidekick.

Is there any technical reason why 17xx compatibility is not possible?
I'm huge GEOS fan, and whatever I get I would be using GEOS with it more that 50% of the time.

**Comment by MC6809E089** [comment]
My SideKick is the Zero 2 variant, and it works great. I think Frenetic said somewhere in another thread that it's not possible for the SideKick to act as an REU, as the PCB doesn't have the correct connections between the cartridge port and the Pi for it to work.

**Comment by frntc** [comment]
As MC6809E089 said, the Zero2W is working fine -- it is the reason for the small PCB that I designed. It's also correct that a Sidekick can not do DMA transfers as the REU does, simply because it would require a different glue logic. The general problem is that the RPi does not have that many GPIOs and one has to design the logic to do either this or that (as long as you want to keep it reasonable simple).
